src/index.js

```diff
@@ -1,92 +1,44 @@
 /* eslint-disable no-labels */
 
-const FS = require('fs');
-const Path = require('path');
-const EventEmitter = require('events');
-
-const { PNG } = require('pngjs');
-const JPEG = require('jpeg-js');
-const BMP = require('bmp-js');
-const GIF = require('omggif');
-const UTIF = require('utif');
-const MIME = require('mime');
-const tinyColor = require('tinycolor2');
-const BigNumber = require('bignumber.js');
-const bMFont = require('load-bmfont');
-const MkDirP = require('mkdirp');
-const rawBody = require('raw-body');
-const fileType = require('file-type');
-const pixelMatch = require('pixelmatch');
-const EXIFParser = require('exif-parser');
-// polyfill Promise for Node < 0.12
-const Promise = global.Promise || require('es6-promise').Promise;
```

Comment:
Removing this means you can drop es6-promise from the dependencies.

Reply:
👍